parm/gfs/monRadSummary.yaml

```diff
@@ -0,0 +1,556 @@
+#
+#  Summary plots for RadMon gfs
+# 
+#    Make 4 plots:
+#      - Obs counts from last 4 cycles
+#      - Avg total bias correction for 1 cycle, 1 day, and 30 days
+#      - Avg omgbc for 1 cycle and 30 days
+#      - Avg penalty for 1 cycle, 1 day, and 30 days
+
+diagnostics:
+
+# Data read
+# ---------
+datasets:
+  - name: summary
+    satellite: g16
+    sensor: abi
+    type: MonDataSpace
+    control_file:
+      - ../data/rad_data/time.abi_g16.ctl
+    filenames:
+      - ../data/rad_data/time.abi_g16.2023070218.ieee_d
```

Comment:
I'm realizing what our first 'infrastructure' tools will be, to generate this list from a script.

Reply:
Yup, that's certainly high on the list.